project/src/components/favorite-offer-card/favorite-offer-card.tsx

```diff
@@ -0,0 +1,40 @@
+import {Offer} from '../../types/offer';
+
+function FavoriteOfferCard(offer: Offer): JSX.Element{
+  const {id, previewImage, price, title} = offer;
```

Comment:
```suggestion
function FavoriteOfferCard(props): JSX.Element{
  const {id, previewImage, price, title} = props.offer;
```
это должен быть компонент, а не просто вызов функции